src/main/java/org/takes/facets/fork/MediaType.java

```diff
@@ -160,7 +161,9 @@ private static String lowPart(final String text) {
      * @return String array with the sectors of the media type.
      */
     private static String[] sectors(final String text) {
-        return new EnglishLowerCase(MediaType.split(text)[0]).string()
+        return new UncheckedText(
+            new Lowered(MediaType.split(text)[0])
```

Comment:
@fanifieiev maybe you can use `Split` here too? (or leave a puzzle)

Reply:
@fabriciofx I will leave puzzle, because we need [this feature](https://github.com/yegor256/cactoos/issues/1251) to do that.